packages/vue/docs/.vuepress/config.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     editLinks: true,
     editLinkText: 'Help us improve this page ❤️',
     nav: [
-      { text: `v${version}`, link:'/migration-guide' },
+      { text: `v${version}`, link:`/introduction/releases/v${version}` },
```

Comment:
releases isn't in introduction :) I think you meant ```/releases/v${version}```

Reply:
yes, thanks.